ratbagd/ratbagd-profile.c

```diff
@@ -325,6 +331,43 @@ static int ratbagd_profile_set_active(sd_bus_message *m,
                                           ratbag_profile_set_active(profile->lib_profile));
 }
 
+static int ratbagd_profile_set_enable(sd_bus_message *m,
+				      void *userdata,
+				      sd_bus_error *error,
+				      bool enabled)
+{
+	struct ratbagd_profile *profile = userdata;
+	int r;
+
+	r = sd_bus_message_read(m, "");
+	if (r < 0)
+		return r;
+
+	(void) sd_bus_emit_signal(sd_bus_message_get_bus(m),
+				  RATBAGD_OBJ_ROOT,
+				  RATBAGD_NAME_ROOT ".Profile",
+				  "EnabledProfileChanged",
```

Comment:
Just checking: is the intended usage here to check for `Enabled` in the callback of this signal? That'd be another round trip over DBus; perhaps we can emit a `ProfileEnabled`/`ProfileDisabled` signal instead.